front/src/components/Mockups.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from "react";
+import "../App.css";
+import axios from 'axios';
+import {Container} from 'reactstrap';
+
+class Mockups extends Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            mockups: []
+        }
+    }
+
+    componentDidMount() {
+        axios.get('http://localhost:8080/mockups/all')
+            .then((response) => {
+                const data = response.data;
+                this.setState({mockups: data});
+            })
+            .catch(() => {
+                console.log("Error");
+            });
+    }
+
+
+
+    render() {
+
+        console.log(this.state.mockups);
```

Comment:
Sprawdzone, działa, więc dodatkowe logi konsoli nie są potrzebne. Po jakimiś czasie trudno będzie określić ich znaczenie.

Reply:
Można ewentualnie dać console.debug, ale też z jakimś dopiskiem co to za log, co on wypisuje